### ISSUE TITLE
Ensure Sidekiq enterprise check happens during Danger run

### ISSUE THREAD

**Issue by cbtkc** [department-of-veterans-affairs/va.gov-team#24460]
Cameron Testerman May 6th at 9:16 AM
Danger PR has been merged! The old Danger step from the Jenkinsfile has been removed and is no longer required. The new Danger GHA results are a required status check before merge now.

Eric Boehs 4 days ago
It doesn't look like the sidekiq enterprise check is working in the new danger. (edited)

Eric Boehs 4 days ago
It fails during the ruby setup step.

Eric Boehs 4 days ago
I think the only solution here is to manually cache the gems and turn off deployment mode.

Eric Boehs 4 days ago
To summarize the above:
SidekiqEnterpriseGaurantor isn't working as ruby/setup-ruby runs bundler with the --deployment flag which fails the build before Danger can even run. The solution appears to be to run bundle install manually in the action (and cache the gems manually).
(edited)
